CppSciComp/CppSciComp/Book.hpp

```diff
@@ -0,0 +1,29 @@
+//
+//  Book.hpp
+//  CppSciComp
+//
+//  Created by Justin Owen on 1/12/19.
+//  Copyright © 2019 Justin Owen. All rights reserved.
+//
+
+#ifndef Book_hpp
+#define Book_hpp
+
+#include <stdio.h>
+#include <string>
+
+class Book{
+public:
+    Book();
+    std::string author, title, publisher, format;
+    int price;
+    void SetYearOfPublication(int year);
+    int GetYearOfPublication() const;
+    
+private:
+    int yearOfPublication;
```

Comment:
when it comes to designing your code, consistency is important in naming conventions. You get to decide how to name variables, classes,functions. There is no right or wrong answer.

There are a variety of cases people use. `snake_case` is typical in python. `CamelCase` is used in java for classes. `camelCase` is used for functions in java also. I personally like `ClassName` camel case for classes, `myFunction` camelCase for functions and `my_variable` for snake case variables.